tools/scripts/seed/certified-user-data.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
   isDataAnalysisPyCertV7: true,
   isMachineLearningPyCertV7: true,
   isRelationalDatabaseCertV8: true,
+  isCollegeAlgebraPyCertV8: false,
```

Comment:
```suggestion
  isCollegeAlgebraPyCertV8: true,
```

I guess this is not forced, but we should add projects to the `completedChallenges` too.